src/components/spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+
+import './Spinner.scss';
+
+interface SpinnerProps {
+  size?: 'small' | 'fullscreen';
+}
+
+export const Spinner: FC<SpinnerProps> = ({ size = 'small' }) => {
+  if (size === 'fullscreen') {
+    return <div className="spinner" />;
+  }
+  if (size === 'small') {
+    return (
+      <div className="spinner-wrapper">
+        <span>Loading...</span>
+        <div className="spinner spinner-small" />
+      </div>
+    );
+  }
+
+  return null;
```

Comment:
Make sure that you really need to return null with your component. If not, try to rewrite your component to avoid returning of null